src/smdba/postgresqlgate.py

```diff
@@ -155,9 +155,7 @@ def estimate(self):
         # No more than 1GB
         self.config['maintenance_work_mem'] = self.to_mb(self.br((mem / 0x10) > megabytes and megabytes or mem / 0x10))
 
-        self.config['checkpoint_segments'] = 8
```

Comment:
The defaults postgresql would choose are based on available RAM and better than our hardcoded values.